sambacc/config.py

```diff
@@ -354,6 +378,25 @@ class DomainGroupEntry(GroupEntry):
     pass
 
 
+class PermissionsConfig:
+    _method_key: str = "method"
+    _status_xattr_key: str = "status_xattr"
+    _default_method: str = "initialize-share-perms"
```

Comment:
I wonder if leaving it to default to the no-op handler is safer. Let the operator explicitly mention that the permission needs to be changed. 

I think it is a good idea to leave the permissions alone if the user has provided a pvc unless the user has explicitly asked for permissions to be changed. When creating a new pvc from a pvc definition we can setup the permissions.

This can also be handled in a new PR.

Reply:
Yeah, I was a bit on the fence about what the default should be. It is very easy for the operator to simply enable the permissions initialization method, so I think may be best to leave it disabled for now.  Let's discuss that in a follow-up PR as you suggest.